js/src/View/TodoListView.js

```diff
@@ -0,0 +1,116 @@
+var Backbone = require('backbone');
+var $ = require('jquery');
+var _ = require('underscore');
+var handlebars = require('handlebars');
+
+var TodoListModel = require('../Model/TodoListModel.js');
+var TodoModel = require('../Model/TodoModel.js');
+var TodoListCollection = require('../Collection/TodoListCollection.js');
+var TodoCollection = require('../Collection/TodoCollection.js');
+var TodoView = require('./TodoView.js')
+
+
+var todoListViewTemplate = require('../templates/todolist.hbs');
+
+
+var TodoListView = Backbone.View.extend({
+    //why does $ syntax not work here?
+
+    tagName: 'div',
+    className: 'todo-list-container',
+
+    events: {
+        'click .deleteList': 'deleteList',
+        'keypress .todo': 'addTodoKeyPress',
+        'click .addTodo': 'addTodoClick',
+        'keypress .listName': 'updateList',
+    },
+
+    initialize: function(){
+        // create todoList model
```

Comment:
I don't think this comment is accurate. A comment on the whole function might also help reduce the need for comments like these inside the function :)